core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -164,6 +164,7 @@ export class CreateServiceComponent implements OnInit {
   public apigeeFeature = this.buildEnvironment.INSTALLER_VARS.feature.apigee && this.buildEnvironment.INSTALLER_VARS.feature.apigee.toString() === "true" ? true : false;
   public selectedDeploymentTarget = "aws_apigateway";
   public azureEnabled: boolean = false;
+  public disableFunction: boolean = false;
```

Comment:
what is the purpose of this variable?

Reply:
since there was no parameter to detect whether functions with NONE are selected or not and now we are NOT creating functions with NONE, that's why this variable is being introduced to take care of that scenario.